src/content/blog/product-brief-3-video-support-better-prerendering--sdk-features.md

```
@@ -60,20 +60,20 @@ Also thanks to the [Puppeteer community](https://github.com/GoogleChrome/puppete
 
 ### Added video support
 
-Because [Microlink API ](/docs/api/getting-started/overview)can detect video, we use the new field to place the streaming source into our preview cards, via our [SDK](/docs/sdk/getting-started/overview).
+Because [Microlink API ](/docs/api/getting-started/overview)can detect video, we use the new field to place the streaming source into our preview cards, via our [SDK](/docs/sdk-legacy/getting-started/overview).
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside link text

(MD039, no-space-in-links)